src/ParallelAlgorithms/Events/Tags.hpp

```diff
@@ -0,0 +1,72 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "Utilities/GetOutput.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace Events::Tags {
+/// \brief The mesh for the observation computational grid. For hybrid methods
+/// like DG-FD the observer mesh changes throughout the evolution.
+template <size_t Dim>
+struct ObserverMesh : db::SimpleTag {
+  using type = ::Mesh<Dim>;
+};
+
+/// \brief Sets the `ObserverMesh` to `domain::Tags::Mesh`
+///
+/// This is what you would use for a single numerical method simulation. Hybrid
+/// methods will supply their own tags.
+template <size_t Dim>
+struct ObserverMeshFromMeshCompute : ObserverMesh<Dim>, db::ComputeTag {
```

Comment:
(optional) `ObserverMeshCompute` would probably be sufficient

Reply:
Done!